### PR TITLE
NO-ISSUE: Adding subsystem-tests for telemeter manifest generation.

### PR DESCRIPTION
**Pull request by ybettan**
In case this is not a production env, the telemeter manifest should be
visible through the manifests API.

Signed-off-by: Yoni Bettan <ybettan@redhat.com>

### CODE REVIEW

File: subsystem/ams_subscriptions_test.go [hunk 1]
```diff
@@ -326,16 +326,6 @@ var _ = Describe("test AMS subscriptions", func() {
 		// ATTENTION: this test override a wiremock stub - DO NOT RUN IN PARALLEL
 		It("UpdateSubscription failed", func() {
 
-			waitForInstallationPreparationCompletionStatus := func(clusterID strfmt.UUID, status string) {
```
Comment (eranco74) [inline subsystem/ams_subscriptions_test.go:329]: How is this relevant to the PR?

Comment (ybettan) [reply subsystem/ams_subscriptions_test.go:329]: I needed to extract it from inside a specific test so another test can use it.
It was a private variable of a specific test and not it is a common function accessible for other subsystem test - specifically for the new test I wrote.

**Review by eranco74 — COMMENTED** [review]
/lgtm
